src/test/java/io/jenkins/plugins/pipeline_log_fluentd_cloudwatch/PipelineBridgeTest.java

```diff
@@ -58,17 +69,25 @@
         configuration.setLogGroupName(logGroupName);
         // TODO reuse form validation when #6 is merged:
         try (Fluency fluency = new Fluency.Builder(new TCPSender.Config().setHost(FluentdLogger.host()).setPort(FluentdLogger.port()).createInstance()).setFlusherConfig(new SyncFlusher.Config().setFlushIntervalMillis(1000)).build()) {
-            fluency.emit("PipelineBridgeTest", Collections.singletonMap("ping", true));
+            fluency.emit(LOG_STREAM_NAME, Collections.singletonMap("ping", true));
             fluency.flush();
         } catch (ConnectException x) {
             assumeNoException("set $FLUENTD_SERVICE_HOST / $FLUENTD_SERVICE_PORT_TCP as needed", x);
         }
+        AWSLogs client = configuration.getAWSLogsClientBuilder().build();
+        try {
+            FilterLogEventsResult events = client.filterLogEvents(
+                    new FilterLogEventsRequest().withLogGroupName(logGroupName).withLogStreamNames(LOG_STREAM_NAME));
+            assertThat("Event didn't reach CloudWatch", events.getEvents(), not(empty()));
```

Comment:
`assumeThat`?

Reply:
no, this means bad configuration in fluentd in the cluster